packages/neos-ui/src/Containers/RightSideBar/style.css

```diff
@@ -2,7 +2,7 @@
     padding-top: 41px;
     padding-bottom: 41px;
     transition: .2s ease transform;
-    will-change: transform;
+    /* will-change: transform; */
```

Comment:
This introduces a bug in Chrome. With this property applied position of fixed elements is buggy. A tooltip with a `left: 500px` will not be offset 500px left from the viewport but from the sidebar.